tck/src/main/java/org/reactivestreams/tck/IdentityProcessorVerification.java

```diff
@@ -415,17 +444,31 @@ public TestSetup apply(Long aLong) throws Throwable {
           final ManualSubscriberWithErrorCollection<T> sub2 = new ManualSubscriberWithErrorCollection<T>(env);
           env.subscribe(processor, sub2);
 
-          sub1.request(1);
-          expectRequest();
-          final T x = sendNextTFromUpstream();
-          expectNextElement(sub1, x);
-          sub1.request(1);
+          final Exception ex = new RuntimeException("Test exception");
 
-          // sub1 has received one element, and has one demand pending
-          // sub2 has not yet requested anything
+          if (doesCoordinatedEmission()) {
+            sub1.request(1);
+            sub2.request(1);
 
-          final Exception ex = new RuntimeException("Test exception");
+            final T x = sendNextTFromUpstream();
+
+            expectNextElement(sub1, x);
+            expectNextElement(sub2, x);
+
+            sub1.request(1);
+            sub2.request(1);
```

Comment:
Shouldn't `expectReuest()` work after these two?

Reply:
Yes, it should. I'll update.